sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -2487,19 +2487,21 @@ object functions {
    */
   def current_timestamp(): Column = withExpr { CurrentTimestamp() }
 
+  // scalastyle:off line.size.limit
   /**
    * Converts a date/timestamp/string to a value of string in the format specified by the date
    * format given by the second argument.
    *
    * A pattern could be for instance `dd.MM.yyyy` and could return a string like '18.03.1993'. All
-   * pattern letters of `java.text.SimpleDateFormat` can be used.
+   * pattern letters of [[https://docs.oracle.com/javase/8/docs/api/java/text/SimpleDateFormat.html java.text.SimpleDateFormat]] can be used.
```

Comment:
We don't generally link to external javadoc, and we've had problems creating a normal javadoc link to the class in many instances. For consistency, I wouldn't bother with this.